From a87ceaac9a2e18593a5dd8f857ddb8bc688c0a53 Mon Sep 17 00:00:00 2001
From: Andre Heider <a.heider@gmail.com>
Date: Tue, 19 Oct 2021 11:08:46 +0200
Subject: [PATCH] tools/pkgconf: add PKG_CONFIG_EXTRAARGS for additional
 arguments

This can be used my the OpenWrt build system to pass additional
arguments when required.

Signed-off-by: Andre Heider <a.heider@gmail.com>
(cherry picked from commit 0e411c69a819bf3c8a122d8421d6c1c0ded33ec8)
---
 tools/pkgconf/files/pkg-config | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/pkgconf/files/pkg-config b/tools/pkgconf/files/pkg-config
index 2c6fd08b4a..91d144d9b1 100755
--- a/tools/pkgconf/files/pkg-config
+++ b/tools/pkgconf/files/pkg-config
@@ -5,4 +5,5 @@ pkg-config.real \
 --keep-system-libs \
 --define-variable=prefix="${STAGING_PREFIX}" \
 --define-variable=exec_prefix="${STAGING_PREFIX}" \
---define-variable=bindir="${STAGING_PREFIX}/bin" "$@"
+--define-variable=bindir="${STAGING_PREFIX}/bin" \
+$PKG_CONFIG_EXTRAARGS "$@"
-- 
2.34.1

